source/release-notes/4.4.txt

```diff
@@ -927,6 +927,16 @@ TLS 1.3 Support
 On CentOS 8 and RHEL 8, MongoDB 4.4 (as well as versions 4.2, 4.0, and
 3.6) support TLS1.3.
 
+User to DN Mapping Exits on Network or Authentication Failures
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:binary:`~bin.mongod` or :binary:`~bin.mongos` returns an error if
+one of the user to Distinguished Name (DN) mappings defined with  
+:setting:`security.ldap.userToDNMapping` can't be evaluated due to
```

Comment:
or the command line options mentioned in the other comment

also contraction

Reply:
updated